D-Upchurch/front-end-capstone
# Added styling

# Description
Added styling to login/register pages.

## Type of change


# Testing Instructions
Clone repo, fetch and check into dcu-dice-roller. Run 'npm start' from root project directory in terminal, in a new tab navigate to the api directory of the project and run the json server. Go to your browser, navigate to the page, check out the styling on the page.
# Checklist:
